replicated-security/pion-1/README.md

```diff
@@ -1,4 +1,4 @@
-# Please use go version 1.19.8 to compile binaries!!
+# Please use go version >1.20 to compile binaries!!
```

Comment:
```suggestion
# Please use go version 1.20 to compile binaries!!
```

If Go 1.20 is being enforced in code, let's remove the implication here that Go 1.20.1, 1.20.2, etc would work